test/Microsoft.Dnx.DesignTimeHost.FunctionalTests/DthStartupTests.cs

```diff
@@ -85,7 +85,7 @@ public void DthStartup_GetProjectInformation(DnxSdk sdk)
             using (var server = DthTestServer.Create(sdk))
             using (var client = new DthTestClient(server))
             {
-                var solution = TestUtils.GetSolution<DthStartupTests>(sdk, "DthTestProjects");
+                var solution = TestProjectsRepository.FindSolution("DthTestProjects");
```

Comment:
The name is ambigous though. It only restore once, instead of at every invocation.